tests/config/test_config_model.py

```diff
@@ -51,6 +57,24 @@ def test_zone_sources(self):
                             assert zone_sources is not None  # pyright type-narrowing
                             self.assertIn(source, zone_sources, f"zone: {zone_key}")
 
+    def test_parser_model_contains_all_parser_data_types(self):
```

Comment:
Should we also test that for Delays?

Reply:
I think the current changes are enough but if you want to add a test feel free to do so.